src/components/video-card/VideoCard.jsx

```diff
@@ -1,34 +1,87 @@
 import React from 'react'
+import axios from 'axios'
 
-
-import { BiLike } from 'react-icons/bi';
+import { AiFillLike,AiOutlineLike } from 'react-icons/ai';
 import {MdOutlineWatchLater} from 'react-icons/md';
 import {HiViewList} from 'react-icons/hi';
+import { useLike } from '../../context/LikesContext';
+import { useState } from 'react';
 
 import './videocard.css'
-const VideoCard = ({vid}) => {
-  const youtubelink = "https://www.youtube.com/embed/"+vid._id
-    console.log(vid);
-    console.log('cardrendered');
+const VideoCard = ({vid,likedvids}) => {
+  const [disabled , setdisabled]=useState(false)
+  
+  const {dispatch,setlikedvids} = useLike();
+  if(likedvids.some((likedvidobj)=> likedvidobj._id===vid._id)){
+    var liketoggle = true
+    
+  }else{
+    console.log('not liked');
+  }
+  
+  // var token = localStorage.getItem('token');
+  var token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI3NzkxNjg3Zi01N2UyLTRkMzQtOTM5Ny01OWUxNGNiOTQ2NTkiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.usQ_8GxoP7_3dkdE3s3yNx23nNsPLi57_7IqOPdlBdM"
```

Comment:
token should not be hardcoded

Reply:
just till I built out auth token will remain hardcoded, will change when auth takes shape.